demo/rmm_plugin/README.rst

```diff
@@ -0,0 +1,51 @@
+Using XGBoost with RAPIDS Memory Manager (RMM) plugin (EXPERIMENTAL)
+====================================================================
+
+[RAPIDS Memory Manager `RMM <https://github.com/rapidsai/rmm>`__ library provides a
```

Comment:
```suggestion
`RAPIDS Memory Manager (RMM) <https://github.com/rapidsai/rmm>`_ library provides a
```

Reply:
This is actually correct, we use double underscore to prevent duplicated names.

Reply:
Okay, you should still remove the square bracket in the beginning

Reply:
Good catch!